examples/train/byteps_dist_gpu_v1beta1.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: "kubeflow.org/v1beta1"
+kind: "MXJob"
+metadata:
+  name: "byteps-mxnet-job"
+spec:
+  jobMode: MXTrain
+  cleanPodPolicy: Running
+  successPolicy: AllWorkers
+  mxReplicaSpecs:
+    Scheduler:
+      replicas: 1
+      restartPolicy: Never
+      template:
+        spec:
+          containers:
+            - name: mxnet
+              image: bytepsimage/mxnet
+              command: ["bpslaunch"]
+    Server:
+      replicas: 2
+      restartPolicy: Never
+      template:
+        spec:
+          containers:
+            - name: mxnet
+              image: bytepsimage/mxnet
+              command: ["bpslaunch"]
+    Worker:
+      replicas: 2
+      restartPolicy: Never
+      template:
+        spec:
+          containers:
+            - name: mxnet
+              image: bytepsimage/mxnet
+              command: ["bpslaunch"]
+              args: ["python3", "/usr/local/byteps/example/mxnet/train_imagenet_byteps.py", "--benchmark", "1", "--batch-size=32"]
+              volumeMounts:
+              - mountPath: /dev/shm
+                name: dshm
+              resources:
+                limits:
+                  nvidia.com/gpu: 8
+          volumes:
+          - name: dshm
```

Comment:
Is this required?

Reply:
Yes, in order to launch BytePS successfully, you need `shm-size`